pkgs/development/libraries/libcyaml/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, libyaml }:
+
+stdenv.mkDerivation rec {
+  pname = "libcyaml";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "tlsa";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0h5ydyqdl8kzh526np3jsi0pm7ks16nh1hjkdsjcd6pacw7y6i6z";
+  };
+
+  buildInputs = [
+    libyaml
+  ];
+
+  installFlags = [ "PREFIX=$(out)" ];
```

* Use `makeFlags` in case the prefix is hardcoded into the files.
* Placeholders are more predictable than make interpolation.

```suggestion
  makeFlags = [ "PREFIX=${placeholder "out"}" ];
```